src/datachain/query/dispatch.py

```diff
@@ -94,7 +94,7 @@ def udf_entrypoint() -> int:
 
 
 def udf_worker_entrypoint() -> int:
-    return get_distributed_class().run_worker()
+    return get_udf_distributor_class().run_worker()
```

Comment:
Now it will use `run_worker` as staticmethod and will not initialize UDF distributor class.